.github/workflows/ci.yml

```diff
@@ -78,8 +78,9 @@ jobs:
 
   # We build a shared docker image called "zoekt". This is not pushed, but is
   # used for creating the indexserver and webserver images.
+  # 
+  # While we run this job on main and PRs, the actual push is only done on the main branch.
   docker:
-    if: github.ref == 'refs/heads/main'
```

Comment:
This is now handled by https://github.com/sourcegraph/zoekt/pull/586/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR149

Reply:
only concern is slowing down PR builds, but I think with the addition of cache-from it should be alright?

Reply:
Yeah, it's going to ensure we reuse layers, it was pretty quick during my tests.

Reply:
It's easy to disable though if we find it slow